prepare-index/src/main/scala/bio/ferlab/fhir/etl/Enrich.scala

```diff
@@ -1,20 +1,13 @@
 package bio.ferlab.fhir.etl
 
-import bio.ferlab.datalake.spark3.SparkApp
+import bio.ferlab.datalake.commons.config.RuntimeETLContext
+import mainargs.{ParserForMethods, arg, main}
 
-object Enrich extends SparkApp {
-  println(s"ARGS: " + args.mkString("[", ", ", "]"))
-  val Array(_, _, jobName, studyIds) = args
+object Enrich {
+  @main
+  def specimen(rc: RuntimeETLContext,
+               @arg(name = "study-ids", short = 's', doc = "Study Ids") studyIds: String): Unit = SpecimenEnricher(rc, studyIds.split(",").toSeq).run()
```

Comment:
En fait je voulais plutôt dire : 
```suggestion
               @arg(name = "study-ids", short = 's', doc = "Study Ids") studyIds: Seq[String]): Unit = SpecimenEnricher(rc, studyIds).run()
```